### PR TITLE
Pen 802 small promo styling

### PR DESCRIPTION
**Pull request by amyerz**
## Description
- update to set small promo to 100% width in various breakpoints
- set paddings

## Jira Ticket
- [PEN-802](https://arcpublishing.atlassian.net/browse/PEN-802)

## Acceptance Criteria
- see comments from https://arcpublishing.atlassian.net/browse/PEN-802?focusedCommentId=475860

## Test Steps
- create new page and add small promo or small manual promo
- test on image left/ right/ top/ bottom to have md-padding (1.5 rem)
- the width of the image should be 100% width 

### Before
_Example: When I clicked the search button, the button turned red._

[include screenshot or gif or link to video, storybook would be awesome]

### After
Screen Shot 2020-11-13 at 9 02 13 PM
Screen Shot 2020-11-13 at 9 03 33 PM

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [ ] Confirmed all the test steps above are working
- [ ] Confirmed there are no linter errors
- [ ] Confirmed this PR has reasonable code coverage
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added.

### CODE REVIEW

File: blocks/small-promo-block/features/small-promo/default.jsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ SmallPromo.propTypes = {
         above: 'Image Above',
         below: 'Image Below',
       },
-    }),
+    }).isRequired,
```
Comment (JackHowa) [inline blocks/small-promo-block/features/small-promo/default.jsx:93]: does arc proptypes use `isRequired` -- haven't seen this before

Comment (JackHowa) [reply blocks/small-promo-block/features/small-promo/default.jsx:93]: don't think you need to include it

**Comment by JackHowa** [comment]
do you have any screenshots for the page published? also it looks like there are a few ways that this could be used with styling not, just based on the admin view.

File: blocks/small-promo-block/features/small-promo/default.jsx (hunk 1 above)
Comment (amyerz) [reply blocks/small-promo-block/features/small-promo/default.jsx:93]: Poorva encountered the drop down with empty "--", which the default should be "Image Right".

**Comment by amyerz** [comment]
> do you have any screenshots for the page published? also it looks like there are a few ways that this could be used with styling not, just based on the admin view.

I don't have a specific published page. I addressed the possible issue about promo items within a chain, and Sara will add a follow up ticket to address those:
https://arcpublishing.atlassian.net/browse/PEN-802?focusedCommentId=478947

File: blocks/small-promo-block/features/small-promo/default.jsx (hunk 1 above)
Comment (JackHowa) [reply blocks/small-promo-block/features/small-promo/default.jsx:93]: huh, so you're saying `isRequired` is needed to have the default render?

**Comment by amyerz** [comment]
@JackHowa  for sample small promo page, the existing prod db homepage should have couple small promo `http://localhost/pf/homepage/?_website=the-sun`, 
Sample to change to `Image left`
Screen Shot 2020-12-02 at 12 21 22 PM

**Review by brentmiller1973 — APPROVED** [review]
Looks good!

**Pull request merged**